package.json

```diff
@@ -18,7 +18,8 @@
     "allowedTokens": "./scripts/allowedTokens",
     "prettier:fix": "npx prettier --write --ignore-unknown .",
     "prettier": "npx prettier --check .",
-    "test": "truffle test --network regtest"
+    "test:truffle": "truffle test --network regtest",
+    "test": "NODE_ENV=test mocha --timeout 10000"
```

Comment:
what this timeout is used for? Is it necessary for all the tests, or do we need it in some specific tests?

Reply:
It is needed specially for unit tests that test the contracts deployment, however I have observed that other tests can reach timeout as well.